### ISSUE TITLE
Mobile app failed to connect to test.sormas

### ISSUE THREAD

**Issue by bernardsilenou** [hzi-braunschweig/SORMAS-Project#7122]
### Bug Description

When a mobile app with A7 was connect to one of the test servers using surveillance officer account, all works fine. However, when connected to another server using the same user role, the app could not authenticate.
### Steps to Reproduce

1. Connect mobile app to https://test-de.sormas.netzlink.com/sormas-ui/ all works fine
2. Connect mobile app to https://test.sormas.netzlink.com/sormas-ui/ app does not connect

### Expected Behavior

- App should be able to connect to server
### System Details

* Device: Adroid 7
* SORMAS version:  1.65.0
* Android version/Browser: 
* Server URL:   https://test.sormas.netzlink.com/sormas-ui/
* User Role: surveillance officer

**Comment by bernardsilenou** [comment]
@MateStrysewske  @fhauptmann 
I thought  both servers have the same configuration.
@fhauptmann Please can you also test and add the logs.

**Comment by MateStrysewske** [comment]
@bernardsilenou I don't think this is an issue that should be put on GitHub, but rather go through Netzlink's ticket system. I can't really imagine it to be a general problem of SORMAS because it's working fine on other systems, and I have no idea where we should look at in order to "fix" it.

**Comment by bernardsilenou** [comment]
@MateStrysewske 
I am just confused, I added it on git because of this issue 
https://github.com/hzi-braunschweig/SORMAS-Project/issues/6896#issuecomment-957603337
These two servers are not dev server? should the dev team not have direct access to the logs?

**Comment by MateStrysewske** [comment]
@bernardsilenou They're not dev servers, they're test instances. We have access to the logs, but I don't expect the logs to be of much help because the error message is pretty clear.

**Comment by bernardsilenou** [comment]
I would followup on it with @fhauptmann
